src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -86,4 +87,183 @@ describe('Test suite for dictionary concepts components', () => {
     wrapper.setProps(nextProps);
     expect(wrapper.instance().props.map_type).toEqual('Associated With');
   });
+
+  it('should handle update of new mappings row source data when isNew is true', () => {
```

Comment:
How are these tests related to execute a search only when ENTER is pressed?

Reply:
This handles the updates of the mappings data, when I changed from the Async Library I changed the way the function(updateAsyncSelectValue) to handle mappings is called in my input  so I had to write a test for it.